### PR TITLE
Document new webhook max_alerts config field

### PR DESCRIPTION
**Pull request by juliusv**
See https://github.com/prometheus/alertmanager/pull/2274

Signed-off-by: Julius Volz <julius.volz@gmail.com>

### CODE REVIEW

**Comment by roidelapluie** [comment]
Can you define uint64 at the top?

**Comment by juliusv** [comment]
@roidelapluie Done. FWIW we also have a bunch of `int` fields in the main Prometheus configuration at https://prometheus.io/docs/alerting/configuration/#configuration-file that we don't define at the top. But I haven't checked yet if they are all different types of ints, etc.

File: content/docs/alerting/configuration.md [hunk 1]
```diff
@@ -46,6 +46,7 @@ Generic placeholders are defined as follows:
 * `<secret>`: a regular string that is a secret, such as a password
 * `<tmpl_string>`: a string which is template-expanded before usage
 * `<tmpl_secret>`: a string which is template-expanded before usage that is a secret
+* `<uint64>`: an unsigned 64-bit integer value
```
Comment (brian-brazil) [inline content/docs/alerting/configuration.md:49]: We use `int` over in the Prometheus config docs.

Comment (juliusv) [reply content/docs/alerting/configuration.md:49]: Yeah, that's what I was wondering about. Should we really just use int everywhere in the docs, or should we also specify it more clearly in the main Prometheus docs? To make sure people don't set values that overflow or underflow.

Comment (brian-brazil) [reply content/docs/alerting/configuration.md:49]: I'd say keep it simple with just int.

Comment (juliusv) [reply content/docs/alerting/configuration.md]: Ok, changed to int.

File: content/docs/alerting/configuration.md [hunk 2]
```diff
@@ -693,6 +699,7 @@ endpoint:
 {
   "version": "4",
   "groupKey": <string>,    // key identifying the group of alerts (e.g. to deduplicate)
```
Comment (brian-brazil) [inline content/docs/alerting/configuration.md:701]: Can we line up the comments?

Comment (juliusv) [reply content/docs/alerting/configuration.md]: Done.

**Pull request merged**